README.md

```diff
@@ -10,13 +10,13 @@ This package provides Julia language bindings for the C++ library [preCICE](http
 
 You can use the Julia bindings for preCICE by adding them as a package in a Julia environment or also directly including the package in a Julia script. For both type of usages you need to have preCICE installed on the system. For preCICE installation you can look at the [installation documentation](https://precice.org/installation-overview.html). 
 
-### Adding the package using the repository link
+### Adding the package
```

Comment:
Lets also keep the option of adding the package using the repository link, especially if the user wants to work with some latest un-released features. Lets have a subsection to the *Adding the package* section where we describe how to add it using the repo link.

Additionally we should also add a few lines on how a user who has already cloned the repo can do the installation. So assume that someone already has the repo on their system and now want to install and use the bindings. Lets provide some instructions in this direction.

Reply:
The repo link is mentioned in the guide to install [custom branches](https://github.com/precice/PreCICE.jl/tree/install-readme#testing-new-features-that-are-on-branches-of-this-repository) and the already cloned bindings are in the guide to install [from a local folder](https://github.com/precice/PreCICE.jl/tree/install-readme#adding-the-package-from-a-local-folder)